### ISSUE TITLE
doc update

### ISSUE THREAD

**Issue by ryw** [astronomerio/astronomer-ee#48]
Yea, it's prob just a sentence or two explaining the options around TLS.

On Sun, Mar 11, 2018 at 8:27 PM, Ry Walker <ry@astronomer.io> wrote:
Is this something worth working into the docs?

On March 9, 2018 at 7:32:09 PM, Greg Neiheisel (greg@astronomer.io) wrote:

You can install without TLS to get a feel for what gets deployed. If you skip / don't set values for global.tlsSecret and global.acme in your config.yaml, it'll get deployed with no certificates. The only challenge here is that docker doesn't like pushing images over insecure connections and will throw some errors if you try to deploy an airflow project from our CLI. 

You could also use the kube-lego project to use free Let's Encrypt certificates. Our installation guide has this documented in more detail. This will get you going without a certificate, or additional costs. You can swap in your own certificate down the line, if you end up getting one.

If you're operating completely within the cluster's network you might be able to disable ingress altogether, by not setting a global.baseDomain value, and directly hit the individual service NodePorts, but that might not be a great long-term fit.

On Fri, Mar 9, 2018 at 6:59 PM, XXX wrote:
Hi Greg,

I do have one immediate question - do have to install with TLS or can we deploy with our service account. We've got two network types, a shared-vpc and more open network. If we're pulling in our shared-vpc, then we'd not require TLS - I can get a cert for the TLS but I'll have to get that before we can install from my NetEng.

**Comment by andscoop** [comment]
The PR I created does not address a customer choosing to deploy a non-public cluster without TLS support enabled.

I don't believe we should recommend a customer does this lightly as we do not currently have support for pushing images to the registry if TLS is disabled. From my understanding this is a limitation of our private registry itself.

Before we include a private platform install w/out TLS enabled in our docs I think we should discuss what that flow looks like. As of right now it would be a bad experience as the CLI would not be functional for deploys and the private registry would be an idle resource on the platform - which is misleading to customers.

@schnie If you agree - let's merge the PR, close this issue and I'll create a separate issue as a placeholder to discuss non-public deploys without TLS enabled. This separate issue(s) would detail

- SPIKE - How does a customer use Astronomer EE as an internal only platform with a private registry and no TLS
- Create a Guide Detailing Non-Public Astronomer EE
  - Under which circumstances a customer might consider this setup (on a VPC)
  - How is this user experience different from a standard Astronomer EE install?
    - What are options to replace the CLI deploy?
    - What other features will be disabled/enabled?
- Disable Docker Private Registry If `tlsSecret` and `loadBalancerIP` are not set

**Comment by schnie** [comment]
@andscoop sounds good. Yea if deployers are on the same network as the internal registry they should be able to deploy without TLS, but they'll need to add the IP of the registry to their local docker engine's `insecure-registries`, so it's kinda hacky, but can work.
